app/assets/stylesheets/custom.css.scss

```diff
@@ -152,3 +152,36 @@ p {
     color: #333;
   }
 }
+
+/* forms */
+
+input, textarea, select, .uneditable-input {
+  border: 1px solid #bbb;
+  width: 100%;
+  margin-bottom: 15px;
+  @include box_sizing;
+}
+
+input {
+  height: auto !important;
+}
+
+label.required:after {
+  color: $purple;
+  content:" *";
+}
+
+#error_explanation {
```

Comment:
Avoid using id selectors<br>Selector `error_explanation` should be written in lowercase with hyphens